makefile.patch

```diff
@@ -0,0 +1,37 @@
+diff --git a/misc/Makefile b/misc/Makefile
+index 714223d10..4ce93069e 100644
+--- a/misc/Makefile
++++ b/misc/Makefile
+@@ -1,5 +1,5 @@
+ PYTHON = python3
+-DESTDIR = /
++DESTDIR = /usr
```

Comment:
We want patches to get upstream and this is simply wrong. `DESTDIR` should be empty by default (not `/`) and there should be a new `PREFIX` variable which would be `/usr/local` by default.